src/php/Lang/Keyword.php

```diff
@@ -11,25 +11,32 @@ final class Keyword extends AbstractType implements IdenticalInterface, FnInterf
 {
     use MetaTrait;
 
+    private ?string $namespace;
     private string $name;
     private int $hash;
 
     /** @var array<string, Keyword> */
     private static $refStore = [];
 
-    public function __construct(string $name)
+    public function __construct(?string $namespace, string $name)
```

Comment:
The constructor is not used anywhere else but in the named constructor inside this class. What about making it private (the constructor)?